.github/workflows/stale-issue-mgr.yaml

```diff
@@ -0,0 +1,31 @@
+# Stale issue handler (could also be extended for stale PRs in future)
+
+# This action will: 
+# * Add a label "stale" on issues after 60 days of inactivity and comment on them if they also have the "external" label
+# * Close the stale issues and pull requests after 7 days of inactivity
+# * If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restartif they have the "external" label
+
+# Runs every Monday morning at 1AM.
+# Docs: https://github.com/actions/stale
+
+name: 'Label and close stale issues'
+on:
+  schedule:
+    - cron: '0 1 * * MON'
+  workflow_dispatch:
```

Comment:
What do we need the dispatch event for?

Reply:
I used it for testing and left it there in case we want to trigger manually.